packages/editor/src/store/selectors.js

```diff
@@ -125,7 +125,8 @@ export const hasNonPostEntityChanges = createRegistrySelector(
 			( entityRecord ) =>
 				entityRecord.kind !== 'postType' ||
 				entityRecord.name !== type ||
-				entityRecord.key !== id
+				entityRecord.key !== id ||
+				entityRecord.hasMetaChanges
```

Comment:
I am not sure if the `hasNonPostEntityChanges` is the proper place to do this. Meta changes are actually part of the post entity, so I find it a bit confusing. I don't know where the selector is used but maybe we need a new one or rename it?

Reply:
Ok I introduced a new function called `hasMetaChanges` in order to make this check independent.